lia-app-uikit/View Controllers/DetailViewController.swift

```diff
@@ -62,4 +102,16 @@ class DetailViewController: UIViewController {
             stackView.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor)
         ])
     }
+    
+    private func loadImage() {
```

Comment:
Pass in imageReferenceId in function parameters instead of getting it from the class declaration